management/src/test/java/org/ehcache/management/providers/settings/EhcacheSettingsProviderTest.java

```diff
@@ -106,7 +104,9 @@ public void test_standalone_ehcache() throws IOException {
         .using(new DefaultPersistenceConfiguration(ROOT.newFolder("test_standalone_ehcache")))
         .using(new DefaultManagementRegistryConfiguration()
             .addConfiguration(statisticsProviderConfiguration)
-            .setCacheManagerAlias("my-cm-1"))
+            .setCacheManagerAlias("my-cm-1")
+            .addTag("boo")
+            .addTags("foo", "baz"))
         .withCache("cache-1", cacheConfiguration1)
         .withCache("cache-2", cacheConfiguration2)
         .build(false);
```

Comment:
This has been discussed with @chrisdennis and @cljohnso : the formatted json format is not compatible with windows and makes the build fail... So we changed it to a 1-liner. The idea is able to pinpoint the exact diff in a line.